benchmark: add description to explain the non-index test result (#5032)
pingcap/docs: benchmark/v4.0-performance-benchmarking-with-tpch.md

## Test result

> **Note:**
>

> > The tables on which SQL statements are executed in this test only has the primary key and does not have secondary index. Therefore, the test result below is not influenced by index.

Reviewer: > In this test, the tables on which SQL statements are executed only have the primary keys and do not have secondary indexes. Therefore, the test result below is not influenced by the index.

> > The tables on which SQL statements are executed in this test only has the primary key and does not have secondary index. Therefore, the test result below is not influenced by index.

Reviewer 2: ```suggestion
> The tables on which SQL statements are executed in this test only have primary keys and do not have secondary indexes. Therefore, the test result below is not influenced by indexes.
```
